subprojects/docs/src/docs/userguide/build_lifecycle.adoc

```diff
@@ -199,15 +169,13 @@ This example uses method `Project.afterEvaluate()` to add a closure which is exe
 
 It is also possible to receive notifications when any project is evaluated. This example performs some custom logging of project evaluation. Notice that the `afterProject` notification is received regardless of whether the project evaluates successfully or fails with an exception.
 
-=== Example: Notifications
-
-[source.multi-language-sample,groovy]
-.build.gradle
-----
-include::{samplesPath}/userguide/buildlifecycle/buildProjectEvaluateEvents/build.gradle[tag=evaluate-events]
-----
+.Notifications
+====
+include::sample[dir="userguide/buildlifecycle/buildProjectEvaluateEvents/groovy",files="build.gradle[tags=evaluate-events]"]
+include::sample[dir="userguide/buildlifecycle/buildProjectEvaluateEvents/kotlin",files="build.gradle.kts[tags=evaluate-events]"]
+====
 
-.Output of **`gradle -q test`**
+.Output of **`gradle -q test`** (partial)
```

Comment:
:x: why not display the full outputs as it was?

If it's about the different file names in the output between groovy/kotlin, then you should have two output files.

Reply:
The examplar tests do normalize absolute paths from the output:

https://github.com/gradle/gradle/blob/bc3f89dc629f64ec37b682341d3d741bfe0cc642/subprojects/integ-test/src/integTest/groovy/org/gradle/integtests/samples/UserGuideSamplesIntegrationTest.groovy#L32-L41

and

https://github.com/gradle/gradle/blob/bc3f89dc629f64ec37b682341d3d741bfe0cc642/subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/logging/SampleOutputNormalizer.groovy#L26-L27

Reply:
You're right, of course. And my diagnostic was wrong. Fixed now.

Reply:
Great, thanks!
The added ` (partial)` on samples titles should then be removed too.
They were other samples outputs truncated in this PR, could you fix them all?

Reply:
Oops, sorry for missing that. Done now.